src/main/java/omero/gateway/BuildConfig.java

```diff
@@ -0,0 +1,7 @@
+package omero.gateway;
+
+public class BuildConfig {
+    public static String getBuildVersion(){
+        return BuildConfig.class.getPackage().getImplementationVersion();
```

Comment:
Looks like a very neat way to introspect a package version. For reference, the way Bio-Formats does that involves some custom code loading the manifest - see https://github.com/openmicroscopy/bioformats/blob/v6.0.0/components/formats-api/src/loci/formats/FormatTools.java#L274.

If this works, this looks definitely more compact and scalable.